sdk/search/search-documents/.eslintrc.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I think these overrides only apply to the samples-dev and samples directories is that right? I am not opposed to this, but maybe a comment (if this flavor of json supports it) explaining that would be great.

Alternatively, I believe if you only apply these to `samples` and not `samples-dev` you may be able to get away with removing the `import/no-extraneous-depdendencies` override?